packages/docusaurus/src/commands/deploy.ts

```diff
@@ -167,12 +167,19 @@ export async function deploy(
           throw new Error('Error: Git push failed');
         } else if (commitResults.code === 0) {
           // The commit might return a non-zero value when site is up to date.
-          const websiteURL =
-            githubHost === 'github.com'
-              ? // gh-pages hosted repo
-                `https://${organizationName}.github.io/${projectName}`
-              : // GitHub enterprise hosting.
-                `https://${githubHost}/pages/${organizationName}/${projectName}`;
+          let websiteURL = '';
+          if (githubHost === 'github.com') {
+            // github.io hosting
```

Comment:
I like this PR, this is a good fix, but let's improve the code a bit (since nested `if` statements it’s hard to understand, we use ternary expression)

```js
websiteURL = projectName.includes('.github.io')
  ?
      `https://${organizationName}.github.io/`;
  :
      `https://${organizationName}.github.io/${projectName}/`;
```

Reply:
Fixed

Reply:
Fixed!